src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -66,6 +68,11 @@
     @Extension
     public static final DescriptorImpl DESCRIPTOR = new DescriptorImpl();
     private static final Logger logger = Logger.getLogger(GhprbTrigger.class.getName());
+
+    private static final ExecutorService pool = Executors.newFixedThreadPool(
+        Integer.parseInt(System.getProperty("GhprbTrigger.poolSize", "5"))
```

Comment:
I recommend:
```java
import jenkins.util.SystemProperties;
...
SystemProperties.getInteger(getClass().getName() + ".poolSize", "5")
```

There's precedent in other parts of the code base like https://github.com/jenkinsci/jenkins/blob/15e69874190ce56e228b823aa1584b8497fc673b/core/src/main/java/hudson/slaves/NodeProvisioner.java#L793